assets/src/application/ui/layout/espressoTable/TableRow.tsx

```diff
@@ -56,15 +45,14 @@ const TableRow = ({
 						}}
 						{...draggableProps}
 						{...dragHandleProps}
-						{...extraProps}
 					>
 						{children}
 					</tr>
 				);
 			}}
 		</Draggable>
 	) : (
-		<tr id={id} className={css} {...extraProps}>
+		<tr id={id} className={css}>
```

Comment:
(falls to knees sobbing) "j...just tell me WHYYYYYYYY????!?"

Reply:
One of the reasons is this: Spread attributes can be useful but they also make it easy to pass unnecessary props to components that don’t care about them or to pass invalid HTML attributes to the DOM. We recommend using this syntax sparingly.

Reply:
I was getting an error that I was passing invalid HTML attributes.

Reply:
The other solution I was thinking of in order to extend was to add props as we need an also type that in order to have control of what we do pass to DOM elements.

Reply:
If you don't like this idea I can pass back spread operator props, track from where invalid attribute was passed and omit that just before passing to HTML element.

Reply:
moved back spread and added safety net

Reply:
thanks for the explanation, the added type safety is great 👍